### ISSUE TITLE
Resolution stuck at 1366X768 with updated script

### ISSUE THREAD

**Issue by H3adcap** [parsec-cloud/Parsec-Cloud-Preparation-Tool#63]
Hello, 

I tried the updated Nvidea script, Parsec does allow bigger resolution but the actual Nvidia driver resolution is still stuck at 1366 x 768, i attached screen shots

So parsec can go higher but the games are still stuck at 1366 x 768, and the pc screen makes black bars on the side if you go a higher resolution 

Any suggestions would be awesome

![PC](https://user-images.githubusercontent.com/20138463/105805244-444e6700-5faa-11eb-8ece-0a7ab527b12f.PNG)
![Parsec Resoltion](https://user-images.githubusercontent.com/20138463/105805286-59c39100-5faa-11eb-85a8-cd3bfefab30e.PNG)
![Nvidea resoltion](https://user-images.githubusercontent.com/20138463/105805325-6e078e00-5faa-11eb-93ed-9daf39f1f1eb.PNG)

**Comment by korjaa** [comment]
I had this exact same issue today. I'm using AWS g4dn.xlarge. For me it was a driver issue and I was able to fix it by installing the AWS driver.

Amazon provided guide is available here:
https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/install-nvidia-driver.html

You want to follow the gaming drivers chapter:
"Option 4: NVIDIA gaming drivers (G4dn instances)"

Your setup might differ, but that's at least how I fixed mine on AWS.
